compiler/record-hessian/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+file(GLOB_RECURSE SOURCES "src/*.cpp")
```

Comment:
you need to include this module to new preset package list to date as of adding like
`infra/packaging/preset/20230907` and
update `infra/debian/compiler/rules`, `infra/packaging/build`,
`infra/debian/compiler/one-compiler.install` files.